pandora-common/src/gameLogic/interactions/_interactionConfig.ts

```diff
@@ -18,6 +18,24 @@ export const INTERACTION_CONFIG = {
 			allowOthers: false,
 		},
 	},
+	useStorageModule: {
+		visibleName: 'Modify contents of storage items',
+		defaultPermissions: {
+			allowOthers: true,
+		},
+	},
+	useLockSlotModule: {
+		visibleName: 'Interact with lock on items',
+		defaultPermissions: {
+			allowOthers: false,
+		},
+	},
+	useTypedModule: {
+		visibleName: 'Change the type of a module on an item',
+		defaultPermissions: {
+			allowOthers: true,
+		},
+	},
```

Comment:
For the names I would somehow emphasize it only applies to _currently worn_ items.
Suggestions:
- Interact with items stored inside worn items
- Change and interact with locks on worn items
- Change the state of worn items
( we never publicly called "typed" module a "typed" module, so I think calling it an item state is a better name)